File: JAVA/Petshop.java
/*Saya Klara Ollivviera Augustine Gunawan dengan NIM 2306205 
mengerjakan soal Tugas Praktikum 1 dalam mata kuliah DPBO 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin */

public class Petshop{
    //atribut yang dibutuhkan
    private String id;
    private String nama;
    private String kategori;
    private int harga;

    //constructor
    public Petshop(){
        this.id = "";
        this.nama = "";
        this.kategori = "";
        this.harga = 0;
    }

    //constructor parameter
    public Petshop(String id, String nama, String kategori, int harga){
        this.id = id;
        this.nama = nama;
        this.kategori = kategori;
        this.harga = harga;
    }

    //getter & setter 
    public String getId(){
        return this.id;
    }

    public void setId(String id){
        this.id = id;
    }

    public String getNama(){
        return this.nama;
    }

    public void setNama(String nama){
        this.nama = nama;
    }
    public String getKategori(){
        return this.kategori;
    }

    public void setKategori(String kategori){
        this.kategori = kategori;
    }

    public int getHarga(){
        return this.harga;
    }

    public void setHarga(int harga){
        this.harga = harga;
    }

}

File: PYTHON/Petshop.py
# Saya Klara Ollivviera Augustine Gunawan dengan NIM 2306205 
# mengerjakan soal Tugas Praktikum 1 dalam mata kuliah DPBO 
# untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin

class Petshop:
    # atribut private
    __id = ""
    __nama = ""
    __kategori = ""
    __harga = 0

    # constructor
    def __init__(self, id = "", nama = "", kategori = "", harga = 0):
        self.__id = id
        self.__nama = nama
        self.__kategori = kategori
        self.__harga = harga

    # getter & setter   
    def getId(self):
        return self.__id
    
    def setId(self, id):
        self.__id = id

    def getNama(self):
        return self.__nama
    
    def setNama(self, nama):
        self.__nama = nama
    
    def getKategori(self):
        return self.__kategori
    
    def setKategori(self, kategori):
        self.__kategori = kategori
    
    def getHarga(self):
        return self.__harga
    
    def setHarga(self, harga):
        self.__harga = harga
        